lib/vm/src/instance/mod.rs

```diff
@@ -1300,21 +1298,6 @@ cfg_if::cfg_if! {
     }
 }
 
-fn check_table_init_bounds(instance: &Instance) -> Result<(), Trap> {
-    let module = Arc::clone(&instance.module);
-    for init in &module.table_initializers {
-        let start = get_table_init_start(init, instance);
-        let table = instance.get_table(init.table_index);
-
-        let size = usize::try_from(table.size()).unwrap();
-        if size < start + init.elements.len() {
-            return Err(Trap::new_from_runtime(TrapCode::TableSetterOutOfBounds));
```

Comment:
Is `TableSetterOutOfBounds` being used somewhere else? If not, can we remove it from the Enum?

Reply:
Yeah, it's still in use! Good call though. I'm assuming I'll get warnings for unused variants / code (though actually not if it's public...). I don't currently see any warnings though